src/components/containers/discoverPage.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { Item, Container } from "semantic-ui-react";
+import PropTypes from "prop-types";
+import { bindActionCreators } from "redux";
+import Article from "../views/article2";
+import * as tagSearching from "../../actions/tagSearchActions";
```

Comment:
I think it's better to have named it tagSearchingActions.

Reply:
@bettblake08 Let me rename it to that.